packages/fx-core/templates/plugins/resource/bot/bicep/botConfiguration.template.bicep

```diff
@@ -1,4 +1,6 @@
+{{#ifCond createNewBotService true}}
```

Comment:
can we see whether `#if` helper from handlebars works? https://handlebarsjs.com/guide/builtin-helpers.html#if

Reply:
Yes. I worked. Let me fix it in next PR.